modules/document_repository/jsx/docIndex.js

```diff
@@ -0,0 +1,195 @@
+import FilterForm from 'FilterForm';
+import {Tabs, TabPane} from 'Tabs';
+import DocUploadForm from './uploadForm';
+import DocCategoryForm from './categoryForm';
+class DocIndex extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoaded: false,
+      filter: {},
+      hiddenHeaders: ['Category', 'Data Dir'],
+    };
+
+    // Bind component instance to custom methods
+    this.fetchData = this.fetchData.bind(this);
+    this.updateFilter = this.updateFilter.bind(this);
+    this.resetFilters = this.resetFilters.bind(this);
+    this.formatColumn = this.formatColumn.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  /**
+   * Retrive data from the provided URL and save it in state
+   * Additionaly add hiddenHeaders to global loris vairable
+   * for easy access by columnFormatter.
+   */
+  fetchData() {
+    $.ajax(this.props.DataURL, {
+      method: 'GET',
+      dataType: 'json',
+      success: function(data) {
+        this.setState({
+          Data: data,
+          isLoaded: true,
+          refresh: 1,
+        });
+      }.bind(this),
+      error: function(error) {
+        console.error(error);
+      },
+    });
+  }
+
+  updateFilter(filter) {
+    this.setState({filter});
+  }
+
+  resetFilters() {
+    this.refs.documentFilter.clearFilter();
```

Comment:
Seems to be a string ref. @zaliqarosli just got rid of them all in #4019, so we should probably make sure we don't have regressions there.

Reply:
done